build.gradle

```diff
@@ -136,10 +136,6 @@ artifacts {
     archives sourceJar
 }
 
-javadoc {
```

Comment:
Please do not remove stuff you did not touch

Reply:
It's doubled in the file